lib/utils.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const crypto = require('crypto')
+
+module.exports = {
+  createHashHex () {
+    const hash = crypto.createHash('sha256')
+    const args = [...arguments]
+    args.forEach((arg) => {
+      hash.update(arg)
```

Comment:
Please include a separator between each argument here, preferably one that won't appear in any of the input strings, such as `'\00'`.  Otherwise, the hash for e.g. `{ ip: "1.2.3.45", email: "test@example.com }` could wind up being the same as the hash for `{ ip: "1.2.3.4", email: "5test@example.com" }`, which would cause us to incorrectly treat those as the same.